test/parallel/test-crypto-keygen.js

```diff
@@ -625,6 +625,22 @@ function convertDERToPEM(label, der) {
     message: 'Invalid ECDH curve name'
   });
 
+  // Test error type when curve is not a string
+  for (const namedCurve of [true, {}, [], 123]) {
+    common.expectsError(() => {
+      generateKeyPairSync('ec', {
+        namedCurve,
+        publicKeyEncoding: { type: 'spki', format: 'pem' },
+        privateKeyEncoding: { type: 'sec1', format: 'pem' }
+      });
+    }, {
+      type: TypeError,
+      code: 'ERR_INVALID_OPT_VALUE',
+      message: `The value "${namedCurve}" is invalid for option ` +
```

Comment:
No need to validate the message.

Reply:
@refack I have to remove the validation of the message or it can be good as is?